readtags-cmd/readtags-cmd.c

```diff
@@ -136,40 +136,59 @@ static void listTags (int pseudoTags)
 static const char *const Usage =
 	"Find tag file entries matching specified names.\n\n"
 	"Usage: \n"
-	"    %s -h\n"
-	"    %s [-ilp] [-n] "
+	"    %s -h | --help\n"
+	"        Print this help message.\n"
 #ifdef READTAGS_DSL
-	"[-Q EXP] "
+	"    %s -HQ| --help-expression filter\n"
+	"        Print available operators in filter expression.\n"
 #endif
-	"[-s[0|1]] [-t file] [-] [name(s)]\n\n"
+	"    %s [OPTIONS] ACTION\n"
+	"        Do the specified action.\n"
+	"Actions:\n"
+	"    -l | --list\n"
+	"        List regular tags.\n"
+	"    [-] name...\n"
+	"        List regular tags matching the name.\n"
```

Comment:
The dots indicate there may be multiple names, right?
```suggestion
	"    [-] name...\n"
	"        List regular tags matching the given names.\n"
```